lib/delivery.server.js

```diff
@@ -146,7 +152,7 @@ Delivery.prototype.subscribe = function() {
 
   //server sending to client
   this.socket.on('send.success', function(uid){
-    _this.pubSub.publish('send.success', uid);
+    _this.pubSub.publish('receive.success', uid);
```

Comment:
I've just tested it and it seems it works, I made a few changes in the example file2server (just added a dot at the beginning of the src paths to js files) and this is the result, but I encourage you to try it also, maybe I'm missing something

![captura de pantalla 2015-05-31 a las 12 20 50](https://cloud.githubusercontent.com/assets/6698645/7901579/886c2f04-078f-11e5-8d24-f4dd49350c79.png)
![captura de pantalla 2015-05-31 a las 12 20 27](https://cloud.githubusercontent.com/assets/6698645/7901580/886c4e94-078f-11e5-8db9-2d02f3e1f78c.png)
![captura de pantalla 2015-05-31 a las 12 20 14](https://cloud.githubusercontent.com/assets/6698645/7901578/886c28e2-078f-11e5-9b51-e18adf32e8ab.png)